src/electionguard/election_polynomial.py

```diff
@@ -19,55 +22,68 @@
 )
 from .schnorr import make_schnorr_proof, SchnorrProof
 
+# TODO: rename to just polynomial and improve the documentation with some links to wikipedia
 
+# TODO: do not use lists here as they are susceptible to order-based attacks
 class ElectionPolynomial(NamedTuple):
     """
-    ElectionPolynomial is a polynomial defined by coefficients. The 0 coefficient is used for a secret key which can 
+    ElectionPolynomial is a polynomial defined by coefficients. 
+    The 0-index coefficient is used for a secret key which can 
     be discovered by a quorum of n guardians corresponding to n coefficients.
     """
 
     coefficients: List[ElementModQ]
+    """
```

Comment:
These can be one liners. I actually disabled the warning for this but we truly should do it
`"""One liner"""`